lib/plugins/aws/package/compile/functions/index.js

```diff
@@ -210,12 +210,32 @@ class AwsCompileFunctions {
         }
       }
 
+      let kmsKeyArn;
       const serviceObj = this.serverless.service.serviceObject;
-      if (functionObject.awsKmsKeyArn || (serviceObj && serviceObj.awsKmsKeyArn)) {
-        const arn = functionObject.awsKmsKeyArn || (serviceObj && serviceObj.awsKmsKeyArn);
+      if (serviceObj && serviceObj.awsKmsKeyArn) {
+        this.serverless._logDeprecation(
+          'SERVICE_DEPRECATED_PROPERTY_AWS_KMS_KEY_ARN',
```

Comment:
Let's not use `DEPRECATED` in deprecation codes (seems verbose, when it's clear we deal with deprecation code)

Also I see we use two different deprecation codes, while we document it under one code.

I think it's ok to use same code for both (which could be: `AWS_KMS_KEY_ARN`), and it's ok that messages are different (however if both will be triggered only first will be shown, still that seems acceptable).

What's important, is that code is used in generated link, so when introducing not documented deprecation code, invalid link is generated